### ISSUE TITLE
Animation error when switching to upgraded weapons

### ISSUE THREAD

**Issue by GeoDash897** [pa1nki113r/Project_Brutality#613]
Hello again! (It's been a while since I posted an issue on here!)

**PB Version**- master (lastest commit 1/21/21)
**GZDoom Version**- 4.5.0

Recently, I had been messing around with the newest master version of PB when I noticed an animation error for upgraded weapons.

If you switch to an upgraded weapon, Doomguy will actually switch to the *previous* version of that weapon (the one without the upgrade) before switching to the *correct* one.

This makes it so that Doomguy switches weapons twice when selecting upgraded weapons- most noticeable with the **LMG upgrade**.

This happens with the **Deagle**, **Quad Shotgun**, **LMG**, and **Pulse Cannon** upgrades.
Basically ones that replace previous weapons.

https://user-images.githubusercontent.com/55932902/105528159-9e9dae00-5cb2-11eb-925a-52274b53be13.mp4

**Comment by JMartinez9820** [comment]
This issue has been known for while, and is a duplicate of #369. Take at that issues and other issues that mention this.

**Comment by GeoDash897** [comment]
Alright. Just close this one then?

**Comment by JMartinez9820** [comment]
I would, yeah. popguy12 did find a way to fix it but he hasn't go to working on it yet.
